### PR TITLE
parquet release notes

### PR DESCRIPTION
**Pull request by galsalomon66**
Signed-off-by: gal salomon <gal.salomon@gmail.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -47,6 +47,17 @@
...
+  using s3select queries.
+  The s3select-engine contains a Parquet-reader(apache/arrow) that enables access
+  to specific columns according to query, which saves a lot of IOPS.
+  The s3select-engine is using (call-back) GetObj-RangeScan to access these types 
+  of objects.
+  The Parquet object is identified by its name(*.parquet) and magic-number exists 
+  in objects. thus, upon sending s3select query, there are 2 main flows, one 
+  for CSV the other for Parquet format.
+  RGW chooses the flow according the object name.
+
```
Comment (cbodley) [inline PendingReleaseNotes:60]: thanks Gal. these release notes are for our users, so this feels a bit heavy on the implementation details

users just need to know that `.parquet` objects are supported for s3 select queries, and that the parquet format can be processed more effeciently than csv

**Comment by galsalomon66** [comment]
@cbodley @mattbenjamin 
can you review it?

**Comment by mattbenjamin** [comment]
@galsalomon66 is this still valid?

**Comment by galsalomon66** [comment]
> @galsalomon66 is this still valid?

the pending-release-notes need a rebase.
nothing had changed in the Parquet execution flow.

**Comment by cbodley** [comment]
> the pending-release-notes need a rebase. (is it relevant?)

good question. these won't show up until the Reef release - better late than never, i guess? up to you

**Comment by galsalomon66** [comment]
> > the pending-release-notes need a rebase. (is it relevant?)
> 
> good question. these won't show up until the Reef release - better late than never, i guess? up to you

@cbodley 
ok, the documentation part is relevant and should reside on upstream documents.

File: PendingReleaseNotes [hunk 2]
```diff
@@ -73,6 +73,102 @@ Relevant tracker: https://tracker.ceph.com/issues/55521
...
+  "--yes-i-really-mean-it" flag. Also, mirroring is expected to be disabled
+  on the file system.
+* MDS upgrades no longer require stopping all standby MDS daemons before
+  upgrading the sole active MDS for a file system.
+
+* Parquet implementation is about accessing columnar objects(Parquet format) 
+  using Parquet reader(apache arrow) that will eventually save a lot of IOPS.
+  The s3select-engine(RGW submodule) contains that Parquet-reader.
+  The Parquet object is identified by its name(*.parquet) and by the magic-number exists 
+  in objects.
```
Comment (cbodley) [inline PendingReleaseNotes:127]: the rebase added back a bunch of other stuff from the quincy release that we don't need now. we just want this Parquet section, and it needs to be under the `>=18.0.0` section at the top

**Review by cbodley — COMMENTED** [review]
please make sure there aren't any merge commits in the final PR

File: PendingReleaseNotes (hunk 2 above)
Comment (galsalomon66) [reply PendingReleaseNotes]: > the rebase added back a bunch of other stuff from the quincy release that we don't need now. we just want this Parquet section, and it needs to be under the `>=18.0.0` section at the top

@cbodley 
I made the changes, should be ok now.

**Comment by galsalomon66** [comment]
jenkins test signed

**Pull request closed without merging**